### ISSUE TITLE
[OS X] Crash during fight.

### ISSUE THREAD

**Issue by Socapex** [ValveSoftware/Dota-2#1756]
Process:               dota_osx [49840]
Path:                  /Users/USER/Library/Application Support/Steam/*/dota_osx
Identifier:            dota_osx
Version:               ???
Code Type:             X86 (Native)
Parent Process:        bash [49837]
Responsible:           steam_osx [49806]
User ID:               501

Date/Time:             2015-05-12 23:02:40.051 -0400
OS Version:            Mac OS X 10.10.3 (14D136)
Report Version:        11
Anonymous UUID:        7FB7401D-CACC-B0E4-83E7-07E82A294D71

Sleep/Wake UUID:       721055FB-139B-4DBA-B033-333E8E2C3CA9

Time Awake Since Boot: 360000 seconds
Time Since Wake:       5700 seconds

Crashed Thread:        3

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000000

VM Regions Near 0:
--> 
    __TEXT                 0000000000064000-0000000000065000 [    4K] r-x/rwx SM=COW  /Users/USER/Library/Application Support/Steam/*

Thread 0:: MainThrd  Dispatch queue: com.apple.main-thread
0   client.dylib                  	0x1d52f88e CInterpolatedVarArrayBase<Vector, false, false>::_Interpolate(Vector*, float, CInterpolatedVarEntryBase<Vector, false> const*, CInterpolatedVarEntryBase<Vector, false> const*) const + 78
1   client.dylib                  	0x1d52fe46 CInterpolatedVarArrayBase<Vector, false, false>::Interpolate(float, float) + 694
2   client.dylib                  	0x1d52bad7 CInterpolatedVarArrayBase<Vector, false, false>::Interpolate(float) + 39
3   client.dylib                  	0x1d5cdc04 C_BaseEntity::BaseInterpolatePart1(float&, Vector&, QAngle&, int&) + 516
4   client.dylib                  	0x1d5b1905 C_BaseAnimating::Interpolate(float) + 485
5   client.dylib                  	0x1d5bfb15 C_BaseAnimatingOverlay::Interpolate(float) + 37
6   client.dylib                  	0x1d5ced70 C_BaseEntity::InterpolateServerEntities() + 1168
7   client.dylib                  	0x1d55a15d OnRenderStart() + 637
8   client.dylib                  	0x1d55c096 CHLClient::FrameStageNotify(ClientFrameStage_t) + 2214
9   engine.dylib                  	0x085cd71a ClientDLL_FrameStageNotify(ClientFrameStage_t) + 202
10  engine.dylib                  	0x086c7b60 SCR_UpdateScreen(bool) + 1232
11  engine.dylib                  	0x086e66e3 _Host_RunFrame_Render() + 1123
12  engine.dylib                  	0x086e9728 _Host_RunFrame(float) + 5560
13  engine.dylib                  	0x086eae1d Host_RunFrame(float) + 877
14  engine.dylib                  	0x087048fb CHostState::State_Run(float) + 347
15  engine.dylib                  	0x08703af8 CHostState::FrameUpdate(float) + 488
16  engine.dylib                  	0x08703905 HostState_Frame(float) + 37
17  engine.dylib                  	0x087d77bf CEngine::Frame() + 2095
18  engine.dylib                  	0x087d3470 CEngineAPI::RunSingleLoop(bool&) + 1584
19  engine.dylib                  	0x087d4be8 CModAppSystemGroup::Main() + 296
20  engine.dylib                  	0x0899a7be CAppSystemGroup::Run() + 46
21  engine.dylib                  	0x087d3cf3 CEngineAPI::RunListenServer() + 147
22  launcher.dylib                	0x00197e5a CSourceAppSystemGroup::Main() + 26
23  launcher.dylib                	0x0019d8de CAppSystemGroup::Run() + 46
24  launcher.dylib                	0x001a182e CSteamApplication::Main() + 62
25  launcher.dylib                	0x0019d8de CAppSystemGroup::Run() + 46
26  launcher.dylib                	0x00198550 LauncherMain + 1072
27  dota_osx                      	0x00064df5 start + 53

Thread 3 Crashed:
0   libsystem_platform.dylib      	0x9acfd1c7 _platform_memmove$VARIANT$sse42 + 215
1   libsystem_c.dylib             	0x9b19f40d __fread + 128
2   libsystem_c.dylib             	0x9b19f379 fread + 49
3   filesystem_stdio.dylib        	0x023b3235 CStdioFile::FS_fread(void*, unsigned long, unsigned long) + 325
4   filesystem_stdio.dylib        	0x023b2878 CFileSystem_Stdio::FS_fread(void*, unsigned long, unsigned long, __sFILE*) + 280
5   filesystem_stdio.dylib        	0x02397998 CFileHandle::Read(void*, int, int) + 104
6   filesystem_stdio.dylib        	0x023977a9 CBaseFileSystem::ReadEx(void*, int, int, void*) + 393
7   filesystem_stdio.dylib        	0x02397613 non-virtual thunk to CBaseFileSystem::Read(void*, int, void*) + 51
8   filesystem_stdio.dylib        	0x023c0743 CPackedStore::ReadData(CPackedStoreFileHandle&, void*, int) + 307
9   filesystem_stdio.dylib        	0x0239796e CFileHandle::Read(void*, int, int) + 62
10  filesystem_stdio.dylib        	0x023977a9 CBaseFileSystem::ReadEx(void*, int, int, void*) + 393
11  filesystem_stdio.dylib        	0x023aa275 CBaseFileSystem::SyncRead(FileAsyncRequest_t const&) + 917
12  filesystem_stdio.dylib        	0x023ad1ad CFileAsyncReadJob::DoExecute() + 93
13  libvstdlib.dylib              	0x008e2bb9 CJob::Execute() + 265
14  libvstdlib.dylib              	0x008e4b17 CJobThread::Run() + 1175
15  libtier0.dylib                	0x003a3cfb CThread::ThreadProc(void*) + 235
16  libsystem_pthread.dylib       	0x93d50e13 _pthread_body + 138
17  libsystem_pthread.dylib       	0x93d50d89 _pthread_start + 162
18  libsystem_pthread.dylib       	0x93d4ee52 thread_start + 34

**Comment by ajinkyapisal** [comment]
Volvo don't care about mac gamers I think. I spend like 100$ on this game and I don't even get a proper game to play

**Comment by Socapex** [comment]
Yes they do, they have fixed dozens of crash I've reported already. They are silent about it, but fixes come and they come fast. Just report the crashes here. 

It is a hate-free place, so I am sure devs are more happy reading bug reports here than on the forums, where everyone is like CryBaby "Volvo don't care about mac gamers I think. I spend like 100$ on this game and I don't even get a proper game to play" yadiyada.

Go back to the forums where you belong, or post good bug reports.

**Comment by ajinkyapisal** [comment]
There is some process that needs to follow when issue is reported. This is simple software engineering concept. Bug reported is still in open state, no action has been taken on the bugs that I have reported in a proper way.

**Comment by Socapex** [comment]
They seem to have prioritized their bug fixing efforts. With Dota 2 Reborn coming, many "annoyances" will be fixed, new ones will come. It would be a waste of dev time to work on non-critical things like that. They do fix crashes, and really fast!

For not marking bugs as fixed, all software companies do this. There isn't much you can do about it. I believe a cleanup of the bug section would be nice when Reborn is out for mac though.

Good day, I know waiting sucks but that is what we have to do :/
